File: Modul-05/SOAL 3/PRAK503-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int MAKSIMAL(int A, int B) {
    return (A > B) ? A : B;
}

int MINIMAL(int A, int B) {
    return (A < B) ? A : B;
}

int main() {
    int batas = 0;
    int maks = -100000;
    int minim = 100000;
    int bilangan;

    scanf("%d", &bilangan);

    while (batas < bilangan) {
        int nilai;
        scanf("%d", &nilai);
        maks = MAKSIMAL(maks, nilai);
        minim = MINIMAL(minim, nilai);
        batas++;
    }

    printf("%d %d", maks, minim);

    return 0;
}

File: Modul-05/SOAL 3/PRAK503-2310817210009-MuhammadIbnuSina.py
def MAKSIMAL(A, B):
    return max(A, B)

def MINIMAL(A, B):
    return min(A, B)

bilangan = int(input())
nilai = list(map(int, input().split()))

maks = -100000
minim = 100000

for nilai in nilai:
    maks = MAKSIMAL(maks, nilai)
    minim = MINIMAL(minim, nilai)

print(f"{maks} {minim}")
